src/config.js

```diff
@@ -25,7 +25,7 @@ const defaultConf = {
         tls: {
             host: process.env.PATHFINDER_TLS_HOST,
             port: process.env.PATHFINDER_TLS_PORT,
-            rejectUnauthorized: process.env.PATHFINDER_REJECT_UNAUTHORIZED.toLowerCase() !== 'false'
+            rejectUnauthorized: process.env.PATHFINDER_REJECT_UNAUTHORIZED && process.env.PATHFINDER_REJECT_UNAUTHORIZED.toLowerCase() !== 'false'
```

Comment:
Same as this, I think:
```suggestion
            rejectUnauthorized: process.env.PATHFINDER_REJECT_UNAUTHORIZED?.toLowerCase() !== 'false'
```

Reply:
Requires newer node version, out of scope for this PR